packages/react/src/components/DataGrid/index.stories.mdx

```diff
@@ -26,6 +27,28 @@ The `rows` props is an array of objects that describe the rows of the table. Eac
 
 We will explore the possibilities that those props provide through the following examples.
 
+## DataList
+
+<Source
+  language='ts'
+  dark
+  format={false}
+  code={`import { DataList } from "@openfun/cunningham-react";`}
+/>
+
+This component is an other wrapper around the more complicated `DataGrid` component. It is made to be used for liste of
```

Comment:
This component is a wrapper around the more complicated `DataGrid` component. It is made to be used for simple lists of elements that don't need pagination nor sorting. Also, it doesn't display any header.